.buildkite/scripts/steps/es_snapshots/build.sh

```diff
@@ -98,7 +98,7 @@ trap 'docker logout docker.elastic.co' EXIT
 docker image push "$KIBANA_ES_CLOUD_IMAGE"
 
 export ELASTICSEARCH_CLOUD_IMAGE="$KIBANA_ES_CLOUD_IMAGE"
-export ELASTICSEARCH_CLOUD_IMAGE_CHECKSUM="$(docker images "$KIBANA_ES_CLOUD_IMAGE" --format "{{.Digest}}")"
+export ELASTICSEARCH_CLOUD_IMAGE_CHECKSUM="$(docker inspect --format='{{index .RepoDigests 0}}' "$KIBANA_ES_CLOUD_IMAGE")"
```

Comment:
I don't have access to `docker.elastic.co/kibana-ci/elasticsearch-cloud`, but when I run `docker inspect --format='{{index .RepoDigests 0}}' "docker.io/library/node:latest"` I see `node@sha256:720d77136dc06bbdac28ef5cd13c385e40a2f1bfaaf7340180fc66820cc184e3`, do we need to reduce this to just ths checksum?